ledger-core/rms/proto_vnvn_validation_vcachedmemoryresponse.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package rms
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+)
+
+func (m *VCachedMemoryResponse) Validate(currentPulse PulseNumber) error {
+	object := m.Object.GetValue()
+	_, err := validSelfScopedGlobalWithPulseBeforeOrEq(object, currentPulse, "Object")
+	if err != nil {
+		return err
+	}
+
+	switch m.CallStatus {
+	case CachedMemoryStateUnknown:
+		if m.Object.IsEmpty() {
```

Comment:
duplicated check

Reply:
moved to another PR https://github.com/insolar/assured-ledger/pull/832/files#diff-27ea9ed0019287b5fd67549774d23164R20